index.js

```diff
@@ -52,32 +63,37 @@ function checkGuess() {
     numberOfGuessesMessage.innerHTML = `You guessed ${guess}. <br> ${remainingAttempts} guesses remaining`;
   }
 
-  if (attempts ==== maxNumberOfAttempts) {
+//  if (attempts ==== maxNumberOfAttempts) { too many =
+  if (attempts === maxNumberOfAttempts) {
     submitButton.disabled = true;
     guessInput.disabled = true;
   }
 
   guessInput.value = '';
-
+  
   resetButton.style.display = '';
 }
 
```

Comment:
In order to allow the user to submit by also pressing "Enter" after typing their number, we can add an additional function here.  The function can be called by an event listener set on the input div.  This function can check if the key pressed is "Enter" or not. If it's "Enter", we can call your checkGuess function.

If you want to try this, make the function, take in the 'event' and check the event's key.

Example:
function checkKeypress(e){
  console.log(e.key === "Enter") 
  if (e.key === "Enter") {
    checkGuess()
  }
}